From: Roland Stigge <stigge@antcom.de>
Date: Mon, 5 Sep 2016 16:23:07 +0100
Subject: include headers

Compile error fix with an additional include line
Fix compile warnings/errors by adding an #include <time.h>
and another for exit().
---
 SwissWatch.c | 1 +
 swisswatch.c | 1 +
 2 files changed, 2 insertions(+)

diff --git a/SwissWatch.c b/SwissWatch.c
index b5e1498..e04a37a 100644
--- a/SwissWatch.c
+++ b/SwissWatch.c
@@ -19,6 +19,7 @@
 #include "Hand.h"
 #include "Mark.h"
 #include <math.h>
+#include <time.h>
 
 #if NeedFunctionPrototypes
 static void Initialize(Widget,Widget,ArgList,Cardinal*);
diff --git a/swisswatch.c b/swisswatch.c
index f70760a..f6391bd 100644
--- a/swisswatch.c
+++ b/swisswatch.c
@@ -11,6 +11,7 @@
 #include <X11/StringDefs.h>
 #include "SwissWatch.h"
 #include <stdio.h>
+#include <stdlib.h>
 
 #if NeedFunctionPrototypes
 static Widget initialize_user_interface(int*argcp,char**argv);
